### ISSUE TITLE
Auchindoun : Sethekk Halls (heroic) issues

### ISSUE THREAD

**Issue by Aetharon** [dalaranwow/dalaran-wow#5045]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Firstly, I would like to bring into attention a recurring problem that I've seen mentioned in issues #3730, #3104, #1866 and #2969. The ID lockdown persists seemingly randomly for up to three days. I have had this issue twice in the last two weeks, three days each.
Secondly, the times I have managed to reach Anzu and summon him, no animation was present. Also, the adds had not behaved properly, only one of the three becoming hostile and the others being unattackable. The bird statues were also nowhere to be found. I will include a video of the expected behaviour.
Thirdly, as of today, there seemed to be another set of issues. The aforementioned animation seemed to trigger, but only partially going through, leaving the summoning altar despawned and Anzu nowhere to be seen. Another issue was that after continuing onwards and killing Talon King Ikiss the final gate towards the exit did not open at all.

**Expected behaviour**: Tell us what should happen instead
The lockdown should reset at 04:59 ST every day as the calendar states. There should be no extension whatsoever.
There are supposed to be three friendly NPC birds spawned when Anzu is summoned - Hawk, Eagle and Falcon Spirit, as part of the  encounter. The Brood of Anzu adds are supposed to attack, all that are spawned, and be attackable by the player.
Thirdly, the gate leading down from Talon King Ikiss should open on his death. I believe that killing Ikiss too quickly makes the trigger stop functioning.

**Steps to reproduce the problem**:

1. Go to Terokkar Forest, Auchindoun, Sethekk Halls on heroic mode
2. Proceed through the dungeon
3. Observe the issues mentioned.

**Include proofs for this behaviour**
https://www.youtube.com/watch?v=LjpBsZ9kf9Q - Proper behaviour of the Anzu encounter.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wotlk.evowow.com/?npc=23035 - Anzu
https://wotlk.evowow.com/?object=185554 - The Raven's Claw, used to summon Anzu
https://wotlk.evowow.com/?npc=18473 - Talon King Ikiss

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/
http://wowwiki.wikia.com/wiki/Anzu
http://wowwiki.wikia.com/wiki/Talon_King_Ikiss

**Comment by robinsch** [comment]
@Lothloryen Is it applied on live yet?

**Comment by Tziva** [comment]
> The aforementioned animation seemed to trigger, but only partially going through, leaving the summoning altar despawned and Anzu nowhere to be seen. 
![wowscrnshot_111017_151259](https://user-images.githubusercontent.com/33547662/32660050-ff035a06-c629-11e7-83c9-6dc12bf7d1d3.jpg)
Confirmed. Anzu's summoning animation was correct(!) after years, but Anzu himself never appeared in the first palce.

**Comment by Lothloryen** [comment]
@robinsch It has indeed been applied, however it looks like something is missing, I might have missed a commit - will check it out.

**Comment by Lothloryen** [comment]
Hello again, it is fine now - enjoy the intro done by @robinsch . Thank you for the reports.

**Comment by Aetharon** [comment]
I would like to confirm that the Anzu encounter now works properly, however the other two issues I have mentioned are still not addressed. Since the opening of this thread, the ID lockdown has not refreshed. Every NPC in the dungeon respawned, including the Raven's Claw. After successfully summoning Anzu one day prior, trying again had only a partial effect. The animation started, then stopped after a few seconds, leaving no trace of the Raven's Claw or Anzu. My guess is that the ID lockdown prevents Anzu being summoned again. As such, there must be a way to enforce the ID refresh. Please, look into this.

**Comment by Aetharon** [comment]
Once again, the ID has not reset. Will I have to open another bug report for this separately? This is a long standing issue and I do not understand the lack of concern.

**Comment by robinsch** [comment]
Did you try to kill something that locks you to the instance? (I think Anzu currently doesnt)

**Comment by Aetharon** [comment]
What I usually do is clear everything inside. I thought it might be something with the corpses remaining unlooted, so I tried to be thorough. However, nothing helped. The lockdown persists randomly. What might be the cause of this?

**Comment by Aetharon** [comment]
The fourth day brought, finally, a refresh to the ID. However, as mentioned earlier, this was the third time in three weeks that the problem had occured. This will most likely happen again, until the cause of this is revealed and addressed.

**Comment by Aetharon** [comment]
Another day, another extension to the ID lockout. Is this in any way intentional?

**Comment by Chenocide** [comment]
@Lothloryen Needs to be reopened, the ID still extends itself.

**Comment by Crispybuilt** [comment]
Random ID extension is general issue with random dungeons/instances.
I've seen it happen on multiple occassions for Karazhan/Zul'Gurub/Sethekk Halls and many other.
It has been present since the release of Kirin Tor.

Will re-open this for now.

**Comment by Aetharon** [comment]
Thank you for reopening this issue. I have one thing to add concerning one of the problems I mentioned in my original report. Ever since Anzu's encounter was properly added, I have had the door leading down from Talon King Ikiss not open at all after Ikiss' defeat. This has happened every time of the four days (of the last 11) I was fortunate enough not to have an ID extend itself.

**Comment by robinsch** [comment]
Sadly I have been sick the last week. I will put some focus on it tomorrow. Any input is helpful.

**Comment by robinsch** [comment]
Ref: https://github.com/dalaranwow/dalaran-wow/issues/5088

**Comment by robinsch** [comment]
Is there something left in this ticket which is not handled by https://github.com/dalaranwow/dalaran-wow/issues/5088?

**Comment by Chenocide** [comment]
Depends on what was missing. As far as I know, it was only the ID extension and the gate not opening.

**Comment by Aetharon** [comment]
I will update the state of the ID resets for the following days. For today, however, I can only say that the gate is still not opening after Ikiss dies.

**Comment by Aetharon** [comment]
For today, the ID has properly reset, which is a welcome change. I will continue to update this thread for the week, concerning this. On the other hand, the part about the door not opening is still pending.

**Comment by Aetharon** [comment]
Three days in a row, the ID has reset properly. The only issue remaining is the door leading down from Talon King Ikiss' chambers. What might or might not be related to the recent fix of non-resetting ID's is that after this raid reset, the ID for Molten Core extended itself for the next week. A similar thing happened for Onyxia's Lair (10 players) last week.

**Comment by Chenocide** [comment]
Just had the reset extend today. Something is still wrong.

**Comment by Aetharon** [comment]
The same thing happened to me. After six days of proper resets, this is quite troubling.

**Comment by robinsch** [comment]
I will just put a workaround in place that you can't extend the ID from heroic instances.

**Comment by robinsch** [comment]
Let's rank this atleast a bit higher. I implemented the extended ID lockout (that can be used from the UI), did a bunch of cleanup and fixed a small mistake in instance reset logic. Since there isn't really much that can be tested on PTR I just request a quick push to Algalon, so FYI @Lothloryen

**Comment by robinsch** [comment]
@Aetharon @Chenocide I have no idea if this really will fix the random instance extend. Just keep me up to date in this ticket and I will take a closer look when the above mentioned change doesn't work out.

**Comment by Aetharon** [comment]
For today, the ID has extended again. I will comment again on this issue after today's fix is pushed to live. One quick reminder about the gate leading down after Talon King Ikiss - that is still not opening properly.

**Comment by robinsch** [comment]
To exit the instance?

**Comment by Aetharon** [comment]
Yes, ever since the Anzu encounter got fixed, the door leading down from Ikiss towards the exit has not opened even once.

**Comment by Aetharon** [comment]
The ID extension happened again today. While the problem seems to occur far less often after the fixes, it persists still.  Also, the minimal issue of the final door leading down from Talon King Ikiss' chambers not opening is still not addressed.

**Comment by Aetharon** [comment]
Once again, the ID extension persisted, making this the third day on the same lockout. Have the recent fixes been reverted?

**Comment by Aetharon** [comment]
And again, the lockdown persists, four days as of now. I will continue to update this the next days, but something must be done...

**Comment by robinsch** [comment]
I have a pending update that could solve this, poke @Lothloryen

**Comment by Lothloryen** [comment]
Hello there,

This should not happen anymore, if it does, let us know and we will take a look but it should be fine already.

Thank you very much for your patience and reports, really appreciated.

**Comment by Lothloryen** [comment]
Hello again,

Corrected the issue with the door.

**Comment by Aetharon** [comment]
Thank you for the assistance, I will keep this updated in case the issues reappear.

**Comment by alexuktc** [comment]
Something struck me today as I was going to Sethekk Halls with my druid. I do not get an ID when I summon Anzu. This means that I can stealth to Anzu, summon the boss with someone else, leave the instance before getting an ID and summon it for someone else. Small issue, but an issue nonetheless.
